.github/workflows/build.yml

```
@@ -0,0 +1,281 @@
+              - 'jobs/cleanup-expired-key-value-records/**'
+
+      - name: Docker metadata
+        if: steps.changes.outputs.cleanup == 'true'
+        id: cleanup-meta
+        uses: docker/metadata-action@v5
+        with:
+          images: europe-docker.pkg.dev/ghost-activitypub/activitypub/cleanup-expired-key-value-records
+          tags: |
+            type=sha
+            type=edge,branch=main
+          cache-from: type=gha
```

actionlint (1.7.7)

input "cache-from" is not defined in action "docker/metadata-action@v5". available inputs are "annotations", "bake-target", "context", "flavor", "github-token", "images", "labels", "sep-annotations", "sep-labels", "sep-tags", "tags"

(action)